yarn/src/main/scala/org/apache/spark/deploy/yarn/ExecutorRunnable.scala

```diff
@@ -290,10 +290,19 @@ class ExecutorRunnable(
       YarnSparkHadoopUtil.setEnvFromInputString(env, userEnvs)
     }
 
+    // lookup appropriate http scheme for container log urls
+    val yarnHttpPolicy = yarnConf.get(
+      YarnConfiguration.YARN_HTTP_POLICY_KEY,
```

Comment:
Do you happen to know when these two fields were added to `YarnConfiguration`? Just want to make sure this isn't introducing an incompatibility with old versions. That said I see this in at least 2.2.0 so probably fine. LGTM.